crates/astria-sequencer/src/app.rs

```diff
@@ -266,6 +269,7 @@ impl App {
     ) -> (Vec<SignedTransaction>, Vec<bytes::Bytes>) {
         let mut signed_txs = Vec::with_capacity(txs.len());
         let mut validated_txs = Vec::with_capacity(txs.len());
+        let mut total_sequenced_data_size: usize = 0;
```

Comment:
Nit: I think I would prefer calling this either `total_sequenced_num_bytes` or `total_sequenced_data_len` just because you construct this from lengths.